client/me/help/help-contact/index.jsx

```diff
@@ -418,12 +421,18 @@ const HelpContact = React.createClass( {
 	 */
 	getView: function() {
 		const { olark, confirmation, sitesInitialized, isSubmitting } = this.state;
+		const { isTicketSupportConfigurationReady, isTicketSupportEligible, ticketSupportRequestError } = this.props;
+
 		const showHappychatVariation = this.shouldUseHappychat();
 		const showChatVariation = olark.isUserEligible && olark.isOperatorAvailable;
-		const showKayakoVariation = ! showChatVariation && ( olark.details.isConversing || olark.isUserEligible );
+		const showKayakoVariation = ! showChatVariation && ( olark.details.isConversing || isTicketSupportEligible );
```

Comment:
Hmm, I'm not sure if that's possible for now, since `olark` is not in the redux store. Or is there a way that I overlooked?